### PR TITLE
Fix indentation for deployment annotation

### PR DESCRIPTION
**Pull request by lazy-eval**
## Release notes

( ) This is not user-visible or is docs only, and no release notes are required.
( ) Release notes are required. Please propose a release note for me.
( ) Release notes are required, with the following suggested text:

```markdown
* 
```

### CODE REVIEW

File: helm/trino-gateway/templates/deployment.yaml [hunk 1]
```diff
@@ -17,7 +17,7 @@ spec:
           # Include the version of trino-gateway-configuration as an input to the deployment checksum. This causes pods to restart on helm update
```
Comment (mosabua) [inline helm/trino-gateway/templates/deployment.yaml:17]: Can you move the docs to also use two space indent. Also can you make the comment line shorter .. maybe 80 char width or so

**Review by mosabua — CHANGES_REQUESTED** [review]

**Comment by mosabua** [comment]
Also .. does this break the chart and should be a release blocker for version 10?

**Review by willmostly — APPROVED** [review]
LGTM

**Review by mosabua — APPROVED** [review]
Looks good now.

**Comment by lazy-eval** [comment]
> Also .. does this break the chart and should be a release blocker for version 10?

As you might have already figured out this is a blocker for deployment

**Pull request merged**